portal/src/texts/core/elevation.md

```diff
@@ -1,3 +1,17 @@
 ## Elevasjon
 
-Vi ønsker at brukerne lett skal se hvilket innhold som er på ett nivå og hvilket som er over- eller underordnet. Til det bruker vi blant annet skygger, som skaper inntrykk av elevasjon. Vi bruker ulike skygger for å skape et hierarki.
+Prinsippet elevasjon handler om å løfte det som er viktig og skape et intuitivt informasjonshierarki. Det kan blant annet gjøres ved å jobbe med nivåer og dybde.
+
+Bruk av skygge indikerer at elementer blir aktivert. Forskyvning ved scroll skaper også dybde og legger viktig innhold høyere opp og nærmere brukeren. Man kan også skape nivåer ved bruk av tydelige størrelsesforhold mellom elementer.
+
+### Nøkkelord
+
+-   Nivåer
+-   Løfte viktig innhold
+-   Tydelig hierarki
+
+### I bruk
+
+Benytt skygger og endring av størrelse for å kommunisere hvilke elementer på siden som har større viktighet. Bruk av størrelsesforhold med høy kontrast styrer fokus mot viktig innhold og bidrar til å skape naturlige brudd eller pauser i oppsettet.
+
+Som designer eller utvikler kan du benytte deg av skalaene for [skygge](), [typografi]() og [sperring]() for å lage konsistente og konsekvente løsninger med kontrast og nivåforskjeller.
```

Comment:
Forslag: Fjern konsistent, evt. bytt ut med et annet ord, som tidligere foreslått, hvis du vil ha to ord. Sett inn en bindestrek etter kontrast- hvis du vil at det skal bety "konrastforskjeller og nivåforskjeller"

Reply:
Tok vekk konsistent. Er meningen at det skal være bare kontrast 😊